backend/readers_backend/readers_backend/templates/index.html

```diff
@@ -5,17 +5,218 @@
     <meta charset="UTF-8">
     <title>Kontinua Foundation Readers</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1>The Sequence</h1>
-
+    <header>
+        <img src="{% static 'kontinua-logo-full.png' %}" alt="Kontinua Foundation" class="logo">
+    </header>
+    <div class="container">
+        {% if latest_us_collection %}
+            <div class="workbook-container">
+                {% for workbook in latest_us_collection.workbooks.all %}
+                    <a class="workbook-link" target="_blank" rel="noopener noreferrer" href="{{ workbook.pdf.url }}">
+                        <img class="workbook-cover" src="{% static 'covers/' %}{{ workbook.number }}.png" alt="Workbook {{ workbook.number }} Cover">
```

Comment:
Use empty alt text. This is decorative and redundant with the following span text.